cranelift/codegen/meta/src/isa/x86.rs

```diff
@@ -194,15 +194,15 @@ fn define_settings(shared: &SettingGroup) -> SettingGroup {
     // Intel Atom CPUs
     let atom = settings.add_preset("atom", "Atom microarchitecture.", preset!(has_ssse3));
     settings.add_preset("bonnell", "Bonnell microarchitecture.", preset!(atom));
-    let silvermont = settings.add_preset(
-        "silvermont",
+    let slm = settings.add_preset(
+        "slm",
```

Comment:
If I'm reading the LLVM tables correctly, it exposes this as _both_ "slm" and "silvermont", but defines them identically. So we should do the same. I think it makes sense to use the longer name "silvermont" as the let-bound preset, and define the "slm" preset in terms of "silvermont".